scheduler/src/cook/mesos/util.clj

```diff
@@ -375,16 +389,32 @@
 (timers/deftimer [cook-mesos scheduler get-running-tasks-duration])
 
 (defn get-running-task-ents
-  "Returns all running task entities"
+  "Returns all running task entities."
   [db]
   (timers/time!
     get-running-tasks-duration
-    (->> (q '[:find ?i
+    (->> (q '[:find [?i ...]
               :in $ [?status ...]
               :where
               [?i :instance/status ?status]]
             db [:instance.status/running :instance.status/unknown])
-         (map (fn [[x]] (d/entity db x))))))
+         (map (partial d/entity db)))))
```

Comment:
(In response to your question below:)
The `find ?i` → `:find [?i ...]` change bought us this cleaner mapping operation here.